pkg/ctrlz/topics/signals.go

```diff
@@ -11,6 +11,8 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//go:build aix || darwin || dragonfly || freebsd || linux || netbsd || openbsd || solaris
```

Comment:
nit: move this before the copyright statement to be consistent with what was done in the other files